### DISCUSSION TITLE
🔘【方法论】我的效率提升方法论 - 任务管理篇 ⑤ GTD 🆕

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#69] [Announcements]
https://imageslr.com/2022/efficiency-gtd.html 

 〇、前言

**Comment by Zting6906** [comment]
最近在研究GTD和PARA的结合，虽然一个是任务管理，一个是信息组织方式，但是两者的切入点都是以可操作的内容为基础，实行“块”的转移。感觉大有可为。

关于博主清单系统的分类那里，个人感觉不用分的太细，GTD的精髓就是轻便简单，分类不同、区别再大的事情最后都会变成一个能不能落实完，执行如何去排序（假如无法做完）。那些复杂具体的分类可以变成PARA中的领域去重点维护。而滴答清单中就简单的变成workspace和life这两块内容即可。我所使用的app不支持指定清单的事项汇总，所以我还会建立一个父级清单来囊括这些执行·项目·等待的事情（如果按照GTD来划分的话），然后再设一个Inbox，一个可能清单，还有一个辅助清单（存放定期回顾之类的事情，辅助GTD流程的完成）

而关于GTD和PARA的结合，我目前还在摸索和学习中。我期望能够把自己的信息管理成系统地嵌入到生活中，实现“无压”的状态。哈哈

**Comment by imageslr** [comment]
> 关于博主清单系统的分类那里，个人感觉不用分的太细，GTD的精髓就是轻便简单

这一点很赞同。我使用的也一直是 4 个大清单（工作、个人、某天、重复）。某些时候，如果一个项目下的子任务较多时，我才会把它拆分成为一个单独的清单，这样便于查看和修改。但是当这个项目完成后，我一定会把清单删除（归档），避免清单系统越来越细。

**Comment by imageslr** [comment]
参考资料，仅作记录，不建议阅读：
* [GTD 心法 - 豆瓣](https://book.douban.com/review/1930909/)：《尽管去做》的读书笔记，简明扼要地概括了全书重点内容，去掉了书中的废话，适合复习看
* [用事件表代替时间表 - 天涯](http://blog.tianya.cn/blogger/post_show.asp?idWriter=0&Key=0&BlogID=588265&PostID=16690142)：作者个人的时间管理心得，其中的几个思路比如关注结果而非过程，很有价值
* [AsianEfficiency OmniFocus 教程 1~10](https://www.asianefficiency.com/omnifocus/)：经典
* [AE OmniFocus 工作流视频教程](https://youtu.be/x7861Lzt6PI)：完整展示收集、处理、回顾等工作流

**Comment by lfscqk67** [comment]
这么简洁方便的滴答被你用得这么复杂
